studio/pages/org/_/[[...routeSlug]].tsx

```diff
@@ -0,0 +1,123 @@
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { NextPage } from 'next'
+import { observer } from 'mobx-react-lite'
+import { Button, IconPlus } from 'ui'
+
+import { withAuth, useStore } from 'hooks'
+import CardButton from 'components/ui/CardButton'
+import ShimmeringCard from 'components/interfaces/Home/ProjectList/ShimmeringCard'
+
+const Header = () => {
+  return (
+    <div className="dark:border-dark border-b p-3">
+      <div className="flex items-center space-x-2">
+        <Link href="/projects">
+          <a>
+            <img
+              src="/img/supabase-logo.svg"
+              alt="Supabase"
+              className="dark:border-dark rounded border p-1 hover:border-white"
+              style={{ height: 24 }}
+            />
+          </a>
+        </Link>
+      </div>
+    </div>
+  )
+}
+
+const GenericOrganizationPage: NextPage = () => {
+  const { app, ui } = useStore()
+  const router = useRouter()
+  const { organizations } = app
+  const { isLoading } = organizations
+
+  const sortedOrganizations = organizations.list()
+  const { routeSlug, ...queryParams } = router.query
+  const queryString =
+    Object.keys(queryParams).length > 0
+      ? new URLSearchParams(queryParams as Record<string, string>).toString()
+      : ''
+
+  const urlRewriterFactory = (slug: string | string[] | undefined) => {
+    return (orgSlug: string) => {
+      if (!Array.isArray(slug)) {
+        return `/org/${orgSlug}/general`
```

Comment:
Is there a reason not to forward on the `queryString` anymore?

Reply:
this particular page doesnt have any support for queryString though - hence why i removed it

Reply:
Yeah why not :) might save us in future if we add something that uses a query string

Reply:
gotcha! just pushed :)